### DISCUSSION TITLE
How do I create crud operation from my existing table

### DISCUSSION THREAD

**Discussion by UmairVarin** [nasirkhan/laravel-starter#253] [Q&A]
Hello, I need to create another modules by keep using (laravel-starter), so when I see code for user model, I am shocked, because too much files there for just one User Model (backend, frontend also), so please tell me the command for another models.

**Comment by nasirkhan** [comment]
I am using [laravel module](https://github.com/nWidart/laravel-modules) package to build and manage modules. But my code structure is a little different than the laravel-module. Currently I am working on to build new package which will be used to manage the modules of the laravel-starter module. By this time you can copy an another module like `Tag` module and rename the necessary file and other settings.

**Reply by rabol** [reply]
Just out of curiosity: how is the structure different and what is the benefit ?

**Comment by UmairVarin** [comment]
Okay dear

**Comment by apvpn** [comment]
Hello @nasirkhan 

Thank you very much for providing such a wonderful module-based laravel starter kit for both frontend and backend in one skeleton.

I installed and reviewed the coding structure.

Could you please provide a specific way to create a new module? like complete CRUD working?

I followed your instructions which you mentioned above to clone the Tags module, however, it is little scary to do that thing because it produces too much errors when any newbie will try to create module for the first time.

So it is my humble request to you to create one wiki about how to create one module with simple title, description etc. fields. so that users can start using it and love more.

I hope you can understand.

Thank you very much.

**Comment by nasirkhan** [comment]
Steps to create a new module is mentioned in the Readme file. Open the following link: https://github.com/nasirkhan/laravel-starter#create-new-module

**Reply by apvpn** [reply]
Thank you very much for the quick response.

Btw, I tried that very initially. I noticed it was giving error on console for the first time.

However, next time it was not giving me error so i used that approach only which you mentioned in the readme file.

In short,  I got the solution for that issue now.
